Locale/enUS/enUS.lua

```diff
@@ -71,7 +71,7 @@ KiwiItemInfo.Locale["enUS"] = {
 	["COMMAND_HELP12"] = "        * |cFFBBBBBBvalue|r -- either true, false, string, or number",
 	["COMMAND_HELP13"] = "    > |cFF888888aci|r -- toggles alwaysCompareItems CVar",
 	["COMMAND_HELP14"] = "    > |cFF888888search|r -- searches through item database for items",
-	["COMMAND_HELP15"] = "        * |cFFBBBBBB${=,>,<}num|r -- show only ilvls 'num' of operation",
+	["COMMAND_HELP15"] = "        * |cFFBBBBBB${=,>,<}num|r -- show only items of ilvl equal, bigger or smaller than 'num'",
```

Comment:
Does this all fit on one line?

Reply:
I'm don't know, what's the character limit?

Reply:
Try and see scenario here.

Reply:
I don't understand what you mean? What is the restriction on the number of characters?